src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameUserInputComboBox.xaml

```diff
@@ -0,0 +1,158 @@
+﻿<ComboBox x:Class="Microsoft.CodeAnalysis.InlineRename.UI.SmartRename.SmartRenameUserInputComboBox"
```

Comment:
A general comment: I see there are some hard code colors in the control, could you verify it won't have any contrast problem in all the VS themes? (dark, light and blue)

Reply:
Yes, I updated the PR description with some screenshots. Please take a look at them. Also, for your information, the theme of this control is copied then modified from the WPF theme we have internally in Visual Studio. The colors uses here are the same in our theme internally.